flowmanager/README.md

```diff
@@ -0,0 +1,27 @@
+# Flow Manager
```

Comment:
Mufasa manages a pride of workflows!  And keeps track of the following resources...